www/src/utils/typography.js

```diff
@@ -123,8 +122,10 @@ const _options = {
       },
       ".gatsby-highlight pre code": {
         display: `block`,
-        fontSize: `95%`,
-        lineHeight: options.baseLineHeight,
+        fontSize: `94%`,
+        lineHeight: 1.5,
```

Comment:
Slightly increased line-height here because IMO this just works better than our regular `1.4`.